Build fails without ONNX_MLIR_ACCELERATORS option after PR#1977

It seems build fails when configuring without `-DONNX_MLIR_ACCELERATORS=NNPA`. I didn't see this before #1977 .

```
CMakeFiles/OMInitAccelerators.dir/InitAccelerators.cpp.o: In function `mlir::Operation::debug_getSuccessors()':
InitAccelerators.cpp:(.text._ZN4mlir9Operation19debug_getSuccessorsEv[_ZN4mlir9Operation19debug_getSuccessorsEv]+0x12): undefined reference to `mlir::SuccessorRange::SuccessorRange(mlir::Operation*)'
collect2: error: ld returned 1 exit status
src/Accelerators/CMakeFiles/OMInitAccelerators.dir/build.make:104: recipe for target 'Release/lib/libOMInitAccelerators.so' failed
make[2]: *** [Release/lib/libOMInitAccelerators.so] Error 1
CMakeFiles/Makefile2:12920: recipe for target 'src/Accelerators/CMakeFiles/OMInitAccelerators.dir/all' failed
make[1]: *** [src/Accelerators/CMakeFiles/OMInitAccelerators.dir/all] Error 2
```

--- a/src/Accelerators/CMakeLists.txt
+++ b/src/Accelerators/CMakeLists.txt
@@ -44,12 +44,12 @@ add_onnx_mlir_library(OMInitAccelerators
   EXCLUDE_FROM_OM_LIBS
 
   DEPENDS 
-    AcceleratorsInc
-    MLIRIR
+    AcceleratorsInc    
 
   LINK_LIBS PUBLIC
     ${ACCEL_LINK_LIST}
     LLVMSupport
+    MLIRIR
   )
 
 add_onnx_mlir_library(OMAccelerator

--- a/src/Compiler/CMakeLists.txt
+++ b/src/Compiler/CMakeLists.txt
@@ -56,9 +56,6 @@ add_onnx_mlir_library(OMCompilerPasses
 
   EXCLUDE_FROM_OM_LIBS
 
-  DEPENDS
-  MLIRIR
-
   INCLUDE_DIRS PRIVATE
   ${FILE_GENERATE_DIR}
 
@@ -69,6 +66,7 @@ add_onnx_mlir_library(OMCompilerPasses
   ${OMLibs}
   OMCompilerOptions
   MLIRAffineTransforms
+  MLIRIR
   MLIRLinalgTransforms
   MLIRLLVMToLLVMIRTranslation
   )
@@ -131,8 +129,7 @@ add_onnx_mlir_library(OMCompilerUtils
   EXCLUDE_FROM_OM_LIBS
 
   DEPENDS
-  ExternalUtil
-  MLIRIR
+  ExternalUtil  
   llc
   opt
 
@@ -148,6 +145,7 @@ add_onnx_mlir_library(OMCompilerUtils
   OMAccelerator
   OMInitAccelerators
   OMVersion
+  MLIRIR
 
   # Link LLVM libraries necessary to query which target architectures
   # are configured.

